doc.html

```diff
@@ -213,8 +215,194 @@ <h1>Alert</h1>
+               style="width: 100%; height: 100%; border:0; transform: scale(1); overflow:hidden;"
+               sandbox="allow-scripts allow-same-origin">
+              </iframe>
+             </code>
+          </div>
+
+          <!---Floating button-->
+          <span class="example">Example of floating Button</span>
+          <p class="comp-example">
+                   Copy paste the below code of floating Button and use it in your own ui.
+          </p>
+          
+         <div class="comp-display-container">
+           <!----- main componentcode start from here ----->
+             <a href="#">
+               <button class="btn primary floating-btn">
+                <i class="fas fa-arrow-up"></i>
+               </button>
+             </a>
+             <a href="#">
+              <button class="btn primary-outline floating-btn">
+               <i class="fas fa-arrow-up"></i>
+              </button>
+            </a>
+           <!----- main componentcode ends from here ----->
+         </div>
+        
+          <div class="code_link">
+             <code>
+              <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%2867%2C196%2C255%2C1%29&t=one-light&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253C%21--primary%2520floating%2520button--%253E%250A%253Ca%2520href%253D%2522%2523%2522%253E%250A%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520primary%2520floating-btn%2522%253E%250A%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-arrow-up%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%253C%252Fbutton%253E%250A%2520%253C%252Fa%253E%250A%253C%21--outline%2520floating%2520button--%253E%250A%2520%253Ca%2520href%253D%2522%2523%2522%253E%250A%2520%2520%253Cbutton%2520class%253D%2522btn%2520primary-outline%2520floating-btn%2522%253E%250A%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-arrow-up%2522%253E%253C%252Fi%253E%250A%2520%2520%253C%252Fbutton%253E%250A%253C%252Fa%253E"
+              style="width:100%; height: 100%; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
i would suggest to avoid inline styling for iframe

Reply:
okay I will create a class for it btw it just gives default so I change only width and height According to my code